tests/test_0_API.py

```diff
@@ -45,5 +45,18 @@ def methods(cls):
     assert (methods(morphio.EndoplasmicReticulum) ==
                  methods(morphio.mut.EndoplasmicReticulum))
 
-    assert (methods(morphio.Mitochondria),
-                 methods(morphio.mut.Mitochondria))
```

Comment:
These two sides are not equivalent, are they?

Reply:
Not at all, that's why I changed the test. Previously the test was not properly written so it was spuriously passing.

Reply:
Spuriously? Apart from `append_root_section` the rest of the methods should be in both, no?
Also why there are the methods listed twice?

Reply:
Look at the new test, all the methods listed in `only_in_mut_mitochondria` are different. I don't know why they are so different for this object though.